src/components/Book.css

```diff
@@ -0,0 +1,240 @@
+.book-container {
+  display: flex;
+  justify-content: space-between;
+  background-color: white;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  padding: 30px;
+  height: 160px;
+  margin-bottom: 20px;
+}
+
+.book-container:last-child {
+  margin: 0;
+}
+
+.book-details {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.category {
+  opacity: 0.5;
+  font-family: sans-serif;
+  font-size: 14px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
```

Comment:
- [ ] There is a lot of CSS rule repetition in this file, please create 1 class that gathers together all the repeated rules and then create more specific classes for when those rules vary a little bit, for example, the class can be named like general-titles-format and there you can place the repeated rules like font family, font-weight, font-stretch, etc, please do it to comply with the [css best practices](https://speckyboy.com/good-bad-css-practices/)
- [ ] You can also make use of classes to store the color #121212 which you are using multiple times as well as the font family sans-serif, please correct this and any other repetition of the same issue all over your css.

Reply:
Issue resolved.